### ISSUE TITLE
[Bug]  skywalking-python get env by not root user, can not load the env

### ISSUE THREAD

**Issue by katelei6** [apache/skywalking#8633]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Python Agent (apache/skywalking-python)

### What happened

In my service, I add the skywalking-python, when I run the server by root, the env config such as SW_AGENT_INSTANCE can get. But when I change the user by others, the env config can not catch. so the ui can not show the instance.

### What you expected to happen

I think the env can get the param by the user config.

### How to reproduce

1 I run the server by user such as xn_test.
![image](https://user-images.githubusercontent.com/79354895/157161742-d062a9a3-2999-4123-a2f6-b26fef74a696.png)
2 the env setting
![image](https://user-images.githubusercontent.com/79354895/157161821-95ec8a15-fe04-449e-88de-51730d373fe2.png)
3 not catch the ip I setted.

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by Superskyyy** [comment]
@katelei6 Hello, please try to execute and check the env vars directly from your shell & python interactively after exporting (in the non-root user).
1. ```printenv SW_AGENT_INSTANCE```
2. ```Python3
	import os
	os.environ['SW_AGENT_INSTANCE']
	```
both cases should show the variable value properly as your IP address.

Please let me know if the outputs are as expected, then we could proceed to investigate further.

**Comment by katelei6** [comment]
1. use printenv SW_AGENT_INSTANCE:  the result is right!
[xn_testdev_ci@host-11-55-250-240 ~]$ printenv SW_AGENT_INSTANCE
11.55.250.240

**Comment by katelei6** [comment]
2、Use Python ， the result is ok！
![image](https://user-images.githubusercontent.com/79354895/158096199-18281c4a-39da-4a2a-ac82-fa74d4340138.png)

**Comment by katelei6** [comment]
3、In the skywalking UI， the result is wrong！
![image](https://user-images.githubusercontent.com/79354895/158096266-7c94c632-de18-44bc-bbbd-2834a016987d.png)

**Comment by katelei6** [comment]
@Superskyyy

**Comment by kezhenxu94** [comment]
I don't recommend to set env var in the bashrc/ zshrc/ profile/ etc., instead, set them in your startup script. Users who run a service in this machine might be surprised to see the service/ instance name is different. Also, these settings should be self-contained in the service (e.g. startup script)

**Comment by wu-sheng** [comment]
Should we still consider this as potential bug or a discussion only?

**Comment by Superskyyy** [comment]
Yes depending on how the user is switched to, a `.bashrc` file may or may not be executed (non-login shell only). 
The current code in the Python agent that reads the env var doesn't look buggy to me? It's the only place that could generate a uuid for the instances.
```
service_instance = os.getenv('SW_AGENT_INSTANCE') or str(uuid.uuid1()).replace('-', '')  # type: str
```
I suppose this is less likely a bug, what you guys think?

**Comment by wu-sheng** [comment]
Moving to discussion.
